flask_lesson_intro/utils.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+
+def get_data():
+    with open("data.json") as file:
```

Comment:
не дуже добре називати змінну `file`, оскільки вона буде так само називатися, як і тип даних `file`
```suggestion
    with open("data.json") as f:
```